### PR TITLE
Replace a few unnecessary uses of Convert.ToString

### PR DESCRIPTION
**Pull request by stephentoub**
Following up on https://github.com/dotnet/runtime/pull/55738#discussion_r670595996

### CODE REVIEW

File: src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/NumberAction.cs [hunk 1]
```diff
@@ -127,20 +127,10 @@ private static string ConvertToArabic(double val, int minLength, int groupSize,
...
-                    return str;
-                }
-                else
-                {
-                    StringBuilder sb = new StringBuilder(minLength);
-                    sb.Append('0', minLength - str.Length);
-                    sb.Append(str);
-                    return sb.ToString();
-                }
+                return str.PadLeft('0', minLength);
```
Comment (stephentoub) [inline src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/NumberAction.cs:133]:
```suggestion
                return str.PadLeft(minLength, '0');
```

File: src/libraries/System.Private.Xml/src/System/Xml/Schema/ContentValidator.cs [hunk 2]
```diff
@@ -876,7 +876,7 @@ sealed class RangeNode : InteriorNode {
 
         public override void Dump(StringBuilder bb, SymbolsDictionary symbols, Positions positions) {
             LeftChild.Dump(bb, symbols, positions);
-            bb.Append($"{{{Convert.ToString(min, NumberFormatInfo.InvariantInfo)}, {Convert.ToString(max, NumberFormatInfo.InvariantInfo)}}}");
+            bb.Append(NumberFormatInfo.InvariantInfo, $"{{{min}, {max}}");
```
Comment (watfordsuzy) [inline src/libraries/System.Private.Xml/src/System/Xml/Schema/ContentValidator.cs:879]: Did escaping of a trailing `}` become optional?
```suggestion
            bb.Append(NumberFormatInfo.InvariantInfo, $"{{{min}, {max}}}");
```

Comment (stephentoub) [reply src/libraries/System.Private.Xml/src/System/Xml/Schema/ContentValidator.cs:879]: > Did escaping of a trailing } become optional?

No :smile:  But this is dead code (behind a compilation constant that nothing it enabling) so nothing caught my typo.  Thanks.

**Comment by danmoseley** [comment]
I think we accidentally missed a review by a committer.

**Pull request merged**